site/docs/v1/tech/apis/scim/index.adoc

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
We'll want to add a version flag somewhere on this page to indicate which version we added SCIM Server Support.

Reply:
done